docs/reference/docs/delete.asciidoc

```diff
@@ -155,7 +155,10 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=refresh]
 
 include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=routing]
 
-include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=timeout]
+`timeout`::
+(Optional, <<time-units, time units>>) Time period to wait for
```

Comment:
This is passed to bulk and has same semantics (except dynamic mapping update and auto-create index does not apply for deletes).